diff --git a/CMakeLists.txt b/CMakeLists.txt
index db8f1da..5e168b3 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -172,10 +172,7 @@ if(COMPILER_HAS_OPTION_OFFLOAD_UNIFORM_BLOCK)
    message("Adding the fno-offload-uniform-block compiler flag")
    add_compile_options(-fno-offload-uniform-block)
 endif()
-if(NOT WIN32 AND ${hip_VERSION_FLAT} GREATER 600140090)
-   message("Adding the enable-post-misched=0 compiler flag")
-   add_compile_options(-mllvm -enable-post-misched=0)
-endif()
+add_compile_options(-mllvm -enable-post-misched=0)
 #
 # Seperate linking jobs from compiling
 # Too many concurrent linking jobs can break the build
